src/rules.js

```diff
@@ -1903,6 +1903,80 @@ module.exports = [
     },
     /**
      ==================================================================
+     TRANSITION
+     ==================================================================
+     */
+    {
+        type: 'pattern',
+        id: 'transition',
+        name: 'Transition',
+        prefix: '.Trs-',
+        properties: ['transition'],
+        allowCustom: true,
+        allowCustomAutoSuffix: false,
+        allowSuffixToValue: true
+    },
+    {
+        type: 'pattern',
+        id: 'transition-delay',
+        name: 'Transition delay',
+        prefix: '.Trsde-',
+        properties: ['transition-delay'],
+        allowCustom: true,
+        allowCustomAutoSuffix: false,
+        allowSuffixToValue: true,
+        rules: [
+            {suffix: 'i', values: ['initial']}
+        ]
+    },
+    {
+        type: 'pattern',
+        id: 'transition-duration',
+        name: 'Transition duration',
+        prefix: '.Trsdu-',
+        properties: ['transition-duration'],
+        allowCustom: true,
+        allowCustomAutoSuffix: false,
+        allowSuffixToValue: true,
+        rules: [
+            {suffix: 'inh', values: ['inherit']}
+        ]
+    },
+    {
+        type: 'pattern',
+        id: 'transition-property',
+        name: 'Transition property',
+        prefix: '.Trsp-',
+        properties: ['transition-property'],
+        allowCustom: true,
+        allowCustomAutoSuffix: false,
+        allowSuffixToValue: true,
```

Comment:
True, but it's going to get ugly really quickly as some of these values could be quite verbose.  And for very short values, it could introduce conflicts with globals and custom suffixes that are unintended.  I don't think we should tackle this right now.